src/test/java/com/hillel/items_exchange/util/PatternHandlerTest.java

```diff
@@ -0,0 +1,304 @@
+        wrongPasswords.add("a1");
+        wrongPasswords.add("aA");
+        wrongPasswords.add("A1");
+        wrongPasswords.add("aa1");
+        wrongPasswords.add("a11");
+        wrongPasswords.add("AA1");
+
+        wrongPasswords.add(" aA1");
+        wrongPasswords.add("aA1 ");
+        wrongPasswords.add(" aA1 ");
+        wrongPasswords.add("a A1");
+
+        wrongPasswords.add("a`1");
+        wrongPasswords.add("A`1");
+        wrongPasswords.add("1#A");
+        wrongPasswords.add("!\"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~");
+
+        wrongPasswords.addAll(getStringsWithRussianOrUkrCharacter("a", "A1"));
+
+        return wrongPasswords;
+    }
+
+    private static List<String> createCorrectPhoneNumbers() {
+        List<String> correctPhoneNumbers = new ArrayList<>();
+        correctPhoneNumbers.add("+38-050-223-32-23");
+        correctPhoneNumbers.add("38-050-223-32-23");
+        correctPhoneNumbers.add("     38  -  050  -  223  -  32  -  23     ");
+        correctPhoneNumbers.add("     38-050  -223-32-23     ");
+        correctPhoneNumbers.add("     38-050-223-   32-23     ");
+
+        correctPhoneNumbers.add("+38.050.223.32.23");
+        correctPhoneNumbers.add("38.050.223.32.23");
+        correctPhoneNumbers.add("     38.050.223.32.23     ");
+
+        correctPhoneNumbers.add("+38-(050)-223-32-23");
+        correctPhoneNumbers.add("+38-(050-223-32-23");
+        correctPhoneNumbers.add("+38-050)-223-32-23");
+
+        correctPhoneNumbers.add("+38-050.223-32-23");
+        correctPhoneNumbers.add("+38-050-223.32-23");
+        correctPhoneNumbers.add("+38-050-223-32.23");
+        correctPhoneNumbers.add("+38-050.223.32.23");
+
+        correctPhoneNumbers.add("+38.050.223-32-23");
+        correctPhoneNumbers.add("+38.050-223.32-23");
+        correctPhoneNumbers.add("+38.050-223-32.23");
+        correctPhoneNumbers.add("+38.050-223-32-23");
+
+        correctPhoneNumbers.add("+00-000-000-00-00");
+
+        return correctPhoneNumbers;
+    }
+
+    private static List<String> createWrongPhoneNumbers() {
+        List<String> wrongPhoneNumbers = new ArrayList<>();
+        wrongPhoneNumbers.add("+3-050-223-32-23");
+        wrongPhoneNumbers.add("+38-50-223-32-23");
+        wrongPhoneNumbers.add("+38-050-22-32-23");
+        wrongPhoneNumbers.add("+38-050-223-3-23");
+        wrongPhoneNumbers.add("+38-050-223-32-2");
+        wrongPhoneNumbers.add("+38-)050-223-32-23");
+        wrongPhoneNumbers.add("+38-050(-223-32-23");
+        wrongPhoneNumbers.add("+38-)050(-223-32-23");
+        wrongPhoneNumbers.add("+38-050-(223)-32-23");
+        wrongPhoneNumbers.add("+38-050-(223)-(32)-(23)");
+
+        wrongPhoneNumbers.add("    +  38-050-223-32-23     ");
+        wrongPhoneNumbers.add("    ++  38-050-223-32-23     ");
+        wrongPhoneNumbers.add("    +  3  8-050-223-32-23     ");
+        wrongPhoneNumbers.add("    +  38-0  50-223-32-23     ");
+        wrongPhoneNumbers.add("    +  38-050-22  3-32-23     ");
+
```

Comment:
Why do we need this empty line?

Reply:
Just for convenience at the time of writing. I forgot to delete it. Thanks.